app/haqq_ante.go

```diff
@@ -21,6 +25,36 @@ func NewHaqqAnteHandlerDecorator(sk keeper.Keeper, h types.AnteHandler) types.An
 			isValid := true
 
 			switch msgs[i].(type) {
+			case *stakingtypes.MsgDelegate, *stakingtypes.MsgCreateValidator:
+				if haqqtypes.IsMainNetwork(ctx.ChainID()) {
```

Comment:
we need testedge1 condition here as well

Reply:
done